### PR TITLE
docs: grammar fixes for statsd.rst

### PR DESCRIPTION
**Pull request by larsks**
this commit fixes some minor grammatic issues with
the statsd daemon documentation.

### CODE REVIEW

File: doc/source/statsd.rst [hunk 1]
```diff
@@ -25,14 +25,14 @@ All the |metrics| will be created dynamically as the |metrics| are sent to
 `gnocchi-statsd`, and attached with the provided name to the |resource| ID you
 configured.
 
-The `gnocchi-statsd` may be scaled, but trade-offs have to been made due to the
+The `gnocchi-statsd` may be scaled, but trade-offs have to be made due to the
```
Comment (chungg) [inline doc/source/statsd.rst]: probably better as "have been" rather than "have to be" as the first part of sentence is past tense.

Comment (larsks) [reply doc/source/statsd.rst]: Good catch.

**Comment by jd** [comment]
Looks like there are some doc8 issues.

**Comment by larsks** [comment]
I wasn't sure. It looked like *everything* was failing, mostly on stuff unrelated to docs changes :)

**Comment by larsks** [comment]
@jd in fact, I don't think there were any `doc8` errors.  The `doc8` logs from travis seem to be failing the same way as everything else, and they run fine on my system.

**Comment by larsks** [comment]
Oh, wait, spotted one!

**Pull request merged**